students/DiannaTingg/lessons/lesson03/assignment/create_customers.py

```diff
@@ -0,0 +1,19 @@
+"""
+Create a customers database for Norton Furniture using peewee, sqlite, and Python.
+"""
+
+import customer_model as cm
+import logging
+
+
+def main():
+    logging.basicConfig(level=logging.INFO)
+    logger = logging.getLogger(__name__)
+
+    logger.info('Creating a database using the customer model.')
+
+    cm.database.create_tables([
+        cm.Customer
+    ])
+
+    cm.database.close()
```

Comment:
I think you want to wait to close the connection until you are done. This might be part of the problem with your tests, you are closing the connection after every transaction, so peewee has to re-establish a connection every time.